### ISSUE TITLE
is stub and send flash finish

### ISSUE THREAD

**Issue by bozzo12** [jeelabs/esp-link#567]
Hello everyone,
I encounter this problem during the flash.
do you know where it could come from?
nothing happens after the flash, but i think it failed
thanks.

hugo
<img width="307" alt="Capture" src="https://github.com/jeelabs/esp-link/assets/96891574/78ea86c7-d071-4fda-85ae-7d6beccc8553">
<img width="956" alt="Capture2" src="https://github.com/jeelabs/esp-link/assets/96891574/73bdfbb9-c506-48c7-aa3a-b4b60c2df3f9">

**Comment by Bear1012061** [comment]
Did you ever find out a fix? Mine is having this same issue!

**Comment by bozzo12** [comment]
yes!! I finally flashed with esp8266flasher, it worked the first time
